File: app/src/main/java/com/example/megaport/mynews/Models/notifications/MyNewsResultAPI.java
package com.example.megaport.mynews.Models.notifications;


import com.example.megaport.mynews.Models.Result;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MyNewsResultAPI {
    @SerializedName("results")
    @Expose
    private final List<Result> results = null;

    public List<Result> getResults() {
        return results;
    }

    //FOR NOTIFICATIONS
    @SerializedName("response")
    @Expose
    private static Response response;

    public static Response getResponse() {
        return response;
    }

}

File: app/src/main/java/com/example/megaport/mynews/Models/notifications/MyNewsResultAPI.kt
package com.example.megaport.mynews.Models.notifications


import com.example.megaport.mynews.Models.Result
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MyNewsResultAPI {

    @SerializedName("results")
    @Expose
    val results: List<Result>? = null

    @SerializedName("response")
    @Expose
    val response: Response? = null

}
